### ISSUE TITLE
Module that can download a new wallpaper everyday.

### ISSUE THREAD

**Issue by sunn-e** [oss2019/pyquotes#33]
I'm creating this issue as a challenge. The module should be able to download a new wallpaper everytime it is run. We can run it as a minimal cronjob or such. 
For ex; brainyquotes has cool wallpapers. You can try using beautiful soup or simple curl expression to download a wallpaper. Seems pretty doable to me. You can take help of [this](https://www.geeksforgeeks.org/implementing-web-scraping-python-beautiful-soup/) :) Connect on gitter if any doubt arrives. Good luck.

**Comment by aayushbisen** [comment]
👋 @sunn-e Can I work on this?

**Comment by sunn-e** [comment]
Of course @aayushbisen

**Comment by harshraj22** [comment]
@sunn-e  I've completed. Let me know how to post

**Comment by sunn-e** [comment]
@harshraj22 , Sent PR?

**Comment by harshraj22** [comment]
@sunn-e  yes. please do review

**Comment by sunn-e** [comment]
can you refer it.

**Comment by harshraj22** [comment]
#39 I don't know if you ment this ?

**Comment by sunn-e** [comment]
Yes, I will keep this open for any new comer. @harshraj22 , you can work on something new.

**Comment by harshraj22** [comment]
@sunn-e  sure. And thanks, so now I can clone other's repo and make changes and add prs ?
(as per gitter message )?

**Comment by sunn-e** [comment]
yes
